### ISSUE TITLE
[Feature Request]could you please add text description of the selectAll button in Table component?

### ISSUE THREAD

**Issue by langlibaitiao** [iview/iview#3464]
### What problem does this feature solve?
 show text description of the selectAll button

### What does the proposed API look like?
I want to show text description of the selectAll button，could you please add it in Table component? i think it is necessary

**Comment by Lovling** [comment]
I agree with you very much. Because I encountered such problems in development, there is no text hint on the selectionAll button of Table component, which brings bad experience to users.
